New hosts UI features (#1736)
theforeman/foreman-documentation: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc

= Editing the System Purpose of a Host

You can edit the system purpose attributes for a {RHEL} host.

> System purpose attributes define which subscriptions to attach automatically to the host.

Reviewer: Auto-attach is only relevant when SCA is turned off.

I would suggest using the text Rich Jerrido gave me for the tooltip:
_System purpose allows you to set the system's intended use on your network and improves the reporting of usage in Subscription Watch._

> System purpose allows you to set the intended use of a system on your network and improves the reporting of usage in Subscription Watch.

Reviewer: Per Jamie Prevatt, this text should now change to
```suggestion
System purpose enables you to set the system's intended use on your network and improves reporting accuracy in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

(sorry for the multiple edits! ping me if you want more context :)

* The host that you want to edit must be registered with the subscription-manager.

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.

> . Click the name of the hosts you want to modify.

Reviewer 2: ```suggestion
. Click the name of the host you want to modify.
```

> . Click *Edit* on the *System purpose* card.

Reviewer 3: ```suggestion
. On the *Overview* tab, click *Edit* on the *System purpose* card.
```

> . Click *Edit* next to the *System purpose* card.

Reviewer 2: ```suggestion
. Click *Edit* on the *System purpose* card.
```

> . Select the purpose attributes for your host.

Reviewer 2: ```suggestion
. Select the system purpose attributes for your host.
```

. Click *Save*.

> // Is there a way to auto-attach subscriptions here?

Reviewer 2: I don't know the answer to this question so you will either have to tag someone specifically or ask an SME.

Author: > I don't know the answer to this question so you will either have to tag someone specifically or ask an SME.

Yeah, I’m adding a few procedure and tagging SME + QA. Thanks for the feedback!

Reviewer: No, there isn't an auto-attach button here, because most users will eventually be using SCA and not need it. Users who need to run auto-attach will need to visit the legacy UI. See also my comment above.
